www/front_src/src/Resources/Details/Header.tsx

```diff
@@ -120,6 +137,19 @@ const Header = ({ details, onSelectParent }: Props): JSX.Element => {
     }
   };
 
+  const navigateToResourceUris = (
+    category: keyof ResourceUris,
+  ): (() => void) => {
+    return (): void => {
+      const url = replaceBasename({
+        endpoint: prop(category, resourceUris) || '',
+        newWord: '/',
+      });
+
+      navigate(`${url}`);
+    };
+  };
+
   if (details === undefined) {
```

Comment:
Do you need to verify `undefined` only ? if not prefer `!details` or `isNil(details)`
```suggestion
  if (!details) {
```